packages/@azure/storage/blob/tests/utils/index.ts

```diff
@@ -109,15 +109,54 @@ export async function createRandomLocalFile(
   });
 }
 
+// Returns a Promise which is completed after the file handle is closed.
+// If Promise is rejected, the reason will be set to the first error raised by either the
+// ReadableStream or the fs.WriteStream.
 export async function readStreamToLocalFile(
   rs: NodeJS.ReadableStream,
   file: string
 ) {
   return new Promise<void>((resolve, reject) => {
     const ws = fs.createWriteStream(file);
+
+    if (process.env.STREAM_DEBUG) {
```

Comment:
Can you please add a comment about `STREAM_DEBUG` ?

Reply:
What sort of comment?  It seems obvious to me what this does, but I'm happy to add whatever comment you think would be helpful.

Reply:
`Populate STREAM_DEBUG in the env to output the console.logs for NodeStream events` ?